ls1intum/Artemis
# Remove duplicate keys from English localization json

Duplicate line pairs: 42/57, 20/59, 21/60, 43/61, 44/62




### Checklist
- [x] No changes to functionality made, just a careful removal of the five duplicate keys.
- [x] Verified that the duplicate keys had the same values associated by rearranging them first.

### Motivation and Context
Remove the unintentional duplicate keys in the `src/main/webapp/i18n/en/global.json` file which had the same value but could cause unexpected behavior if changes are made to the wrong one.

### Description
Remove the line with the higher number for each of the duplicate line pairs 42/57, 20/59, 21/60, 43/61, 44/62.

### Steps for Testing
Check if the English localization in Artemis for `fileUploadSubmission`, `programmingSubmission`, `textSubmission`, `fileUploadExercise` and `textExercise` did not change.
